images/win/scripts/Installers/Install-CommonUtils.ps1

```diff
@@ -0,0 +1,13 @@
+$commonPackages = (Get-ToolsetContent).choco.common_packages
+
+foreach ($package in $commonPackages)
+{
+    Write-Host "Installing ${package.name}"
+    if ($package.args) {
```

Comment:
Maybe we can get rid of it and pass $package.args anyway. Choco-install will skip them if the variable are empty